app/client/packages/design-system/widgets/src/components/FieldDescription/src/FieldDescription.tsx

```
@@ -1,16 +1,18 @@
-import type { FieldDescriptionProps } from "./types";
+import { Text } from "@appsmith/wds";
+import { Text as AriaText } from "react-aria-components";
+import type { TextProps as AriaTextProps } from "react-aria-components";
 
-export const FieldDescription = (props: FieldDescriptionProps) => {
-  const { description, isInvalid } = props;
+export const FieldDescription = (props: AriaTextProps) => {
+  const { children } = props;
 
-  if (!Boolean(description) || Boolean(isInvalid)) return null;
+  if (!Boolean(children)) return null;
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)